conductor-client/src/main/kotlin/com/openlattice/organizations/OrganizationMetadataEntitySetsService.kt

```diff
@@ -334,17 +382,6 @@ class OrganizationMetadataEntitySetsService(
                 columnEntities,
                 columnAuthorizedPropertyTypes
         )
-
-        val datasetEntityKeyId = getDatasetEntityKeyId(organizationMetadataEntitySetIds, table.id)
-        val datasetColumnEntity = mutableMapOf<UUID, Set<Any>>(
-                propertyTypes.getValue(COL_INFO).id to setOf(mapper.writeValueAsString(columnEntities))
-        )
-
```

Comment:
what's the reason for deleting this chunk?

Reply:
`ol.columninfo` is no longer needed so it doesn't need to exist.

Reply:
so are we not supporting keyword searches executing across tables + columns anymore?

Reply:
Keyword search still works for tables and columns. Are you asking if we are supporting a single search request for both tables and columns at the same time?

Reply:
yeah, seems like at the very least this will break paging.